<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-9M55QQJ" failures="1" tests="2" name="Test" time="0.072" errors="0" timestamp="2022-12-31T23:33:00 IST">
  <testcase classname="DisadvantagesOfHardAssert.Disadvantage" name="Testing" time="0.014">
    <failure type="java.lang.AssertionError" message="a and b are equal TC is failed did not expect [pune] but found [pune]">
      <![CDATA[java.lang.AssertionError: a and b are equal TC is failed did not expect [pune] but found [pune]
at DisadvantagesOfHardAssert.Disadvantage.Testing(Disadvantage.java:18)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- Testing -->
  <testcase classname="DisadvantagesOfHardAssert.Disadvantage" name="Testing1" time="0.001"/>
</testsuite> <!-- Test -->
